### DISCUSSION TITLE
ZIA Feature for Authentication Settings

### DISCUSSION THREAD

**Discussion by theIncredibleFoo** [zscaler/terraform-provider-zia#133] [Q&A]
We are using ZIA with manual configuration for all tasks so far. Hence we were delighted to see there is a Terraform provider for basic configuration tasks. However, to be used in our Business context we would need features for Authentication Settings, NSS and Proxy PAC Settings (details see below). Is there any roadmap for the ZIA provider that will add these features? 

**1. Authentication Settings** 
- Create SAML based Authentication 
- Create Identity Provider for Azure Active Directory 

**2. NSS**
- Create NSS Server 
- Create NSS Feed

**3. Proxy Pac**
- Upload a Proxy PAC File

**Comment by willguibr** [comment]
Hi @theIncredibleFoo Thanks for reaching out and thanks for the feedback.
At the moment, there are no plans to include these capabilities in the provider, simply because we don't have public APIs exposed for the purposes you've described.
Can you please email me at wguilherme@you_know_the_rest.com ;) ? I'll be happy to set up some time for us to discuss so I can understand your use cases a bit more in depth.
BTW: Apologies for the delay to respond.
